graph-ts/chain/tendermint/proto/src/t.v1.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+
+package t.v1;
```

Comment:
Can't we use the original files here? We won't be using any custom generator here and `as-proto` should handle the imports.

This can be done as a part of [GRAPH-305](https://figmentio.atlassian.net/browse/GRAPH-305).

Reply:
I'll resolve this and we can discuss for applying this in GRAPH-305. This is not the same file as fig/tendermint/codec/v1/codec.proto since this one import cosmos type and the other have them included in it.